### PR TITLE
Remove `future` package dependency

### PR DESCRIPTION
**Pull request by fruch**
recently CVE-2022-40899 was found on latest future package

* removes support for python2

Fixes: #40

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py36, py37, py38, py39, pep8, lint
+envlist = py36, py37, py38, py39, py310, py311, pep8, lint
```
Comment (asherf) [inline tox.ini:2]: nice. thanks!

**Comment by fruch** [comment]
> Thank you, we need to update the readme to mention that we're no longer supporting python 2.7 I will merge this for now and get the readme later

thanks you for the quick merge,

if you could give me shout when it get a new version in pypi, it would be nice.

**Comment by ahothan** [comment]
Was about to release a new version when I saw pylint now has a lot of new warnings,
I need to see if they can be fixed prior to the release.

**Comment by ahothan** [comment]
finished fixing all pylint warnings
will push the changes and apply new tag next

**Comment by ahothan** [comment]
0.10.1 finally pushed to PyPI after migrating to Github CI.
Thank you

**Pull request merged**